vite-project/src/components/Navbar.css

```diff
@@ -0,0 +1,37 @@
+.Navbar {
```

Comment:
Benefit of using a front end library like bootstrap is you dont have to write your own css, this also applies to the grid you made:

https://getbootstrap.com/docs/5.0/components/navbar/

Not that spending time learning css isnt useful!